templates/vue/src/utils/constants.js

```diff
@@ -30,3 +30,11 @@ export const licenses = {
 }
 
 export const DEFAULT_DEPTH = 3
+
+export const nodeStatuses = {
+  PUBLISH: "publish",
+  REJECT: "reject",
+  ACCEPT: "accept",
+  SUBMITTED: "submitted",
+  DRAFT: "draft",
+}
```

Comment:
a) this is never used anywhere
b) if you are planning to use this, please split it into `nodeStatuses` and `reviewStatuses`

Reply:
Deleted (might be a good idea in the future though?)